src/renderer/components/UI/Config.vue

```diff
@@ -36,21 +36,18 @@
         {{ $t("app.config.wagoAccount" /* Wago Account (optional) */) }}
       </p>
       <input type="text" v-model="wagoUsername" size="11" />
-      <v-button @click="config.wagoUsername = wagoUsername">{{
-        $t("app.config.ok" /* Ok */)
-      }}</v-button>
+      <v-button @click="config.wagoUsername = wagoUsername">Ok</v-button>
       <img
         v-if="config.wagoUsername"
         class="green"
-        :src="require(`@/assets/ok.png`)"
+        :src="require(`@/assets/ok-icon.png`)"
```

Comment:
don't revert 94209afc30be87ac2549898bbd07a293f17c2e4a changes

Reply:
Fixed all icon paths in latest commit